### DISCUSSION TITLE
Bring back translations

### DISCUSSION THREAD

**Discussion by softorangetech1122** [community/community#33984] [Other Feature Feedback, Questions, & Ideas]
I've seen this edit on the GitHub Blog -- https://github.blog/2010-07-13-github-in-your-language/#:~:text=July%2013%2C%202010-,Edit%20(11/18/2016)%3A%20GitHub%20no%20longer%20supports%20languages%20other%20than%20English

This may pose some issues with GitHub worldwide accessibility. It might be better to **bring back translations** to make GitHub appeal more internationally.

**Comment by benjiallen** [comment]
@softorangetech1122 thanks for the feedback. I agree, this would be a cool feature.

**Comment by linkfang** [comment]
Can not agree more! 

Language should not be the barrier here. If GitHub could remove it, some great libraries/projects can help more developers/projects which means even more end users will benefit from it. 

And if possible, it would be great if the translate stuff could be configured on user level, which means, all repos the user visits will be auto translated into the configured language by default. This feels more natural and helpful, compared to it requires user to click translate button on every issue or even post inside the issue.
